src/odemis/gui/cont/tabs.py

```diff
@@ -3716,51 +3719,29 @@ def _onClickFocus(self, evt):
         self.tab_data_model.autofocus_active.value = not self.tab_data_model.autofocus_active.value
 
     def _onManualFocus(self, event):
-        """Called when manual focus btn receives an event."""
-        # TODO need to handle all cases where focus is enabled/used
+        """
+        Called when manual focus btn receives an event.
+        """
         main = self.tab_data_model.main
         bl = main.brightlight
         align_mode = self.tab_data_model.align_mode.value
-
         if event.GetEventObject().GetValue():  # manual focus btn active
-            # switch on calibration light
-            # close slit spectrograph maximal
+            # # Disable autofocus if manual focus btn pushed
+            #         # self.panel.btn_autofocus.Enable(False)
+            #         # self.panel.gauge_autofocus.Enable(False)
             if align_mode == "streak-align":
-                s = None
-                opath = "streak-focus"
+                s = []
+                self.spectrograph.moveAbsSync({"wavelength": 0})
```

Comment:
I don't think setting wavelength to 0 is needed, as the OPM should take care of it. Or is there a special reason?

Reply:
I can remove it, I just left it because it was already there.